docs/user-guide/common-use-cases/shared-wallets.md

```diff
@@ -0,0 +1,447 @@
+---
+order: 10
+title: Shared wallets
+---
+
+## Pre-requisites
+ - [[how-to-start-wallet-server]]
+  - In order to be able to send transactions, our wallet must have funds. In case of `preview` and `preprod` [testnets](https://testnets.cardano.org/en/testnets/cardano/overview/) we can request tADA from the [faucet](https://testnets.cardano.org/en/testnets/cardano/tools/faucet/).
+
+## Overview
+This guide is to show how to create a shared wallet and make a shared transaction providing witnesses from all required co-signers required by `payment_script_template` and `delegation_script_template`. In this example we will create two shared wallets using the same template for both payment and delegation operations. The template that we use indicates that we require all signatures from co-owners in order to make a transaction. In our case the wallet will have two co-owners `cosigner#0` and `cosigner#1`:
+```
+"template":
+    { "all":
+       [ "cosigner#0", "cosigner#1"]
+    }
+```
+
+## Creating wallets
+First let's create two `incomplete` shared wallets using [`POST /shared-wallets`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postSharedWallet) encpoints. The wallets are called "incomplete" because they do not have full information about all cosigners yet.
```

Comment:
```suggestion
First let's create two `incomplete` shared wallets using [`POST /shared-wallets`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postSharedWallet) endpoint. The wallets are called "incomplete" because they do not have public keys from all cosigners yet.
```